vuepress/docs/v6.3.2/tutorials/devops/change-default-datasource.md

```diff
@@ -35,7 +35,7 @@ may have unintended consequences.
 #### 1.2. Any modules that may be required for your datasource or other connection resource
 If the resource you need to connect to requires some custom classes, such as JDBC drivers, please add these as  
 a module to Wildfly/EAP. As an example, you can look at our 
```

Comment:
Let's remove the "As an example" sentence plus the following sentence. It's not an equivalent resource for the example here and I don't see an obvious one off hand.